tools/git/hooks/src/pre_commit.py

```diff
@@ -121,6 +121,8 @@ def is_checked_file_pattern(file_name):
 
 def is_ignored_file_pattern(file_name):
     ignored_patterns = [
+        r"FreeRTOS-Plus",
```

Comment:
Why is the entire directory of `FreeRTOS-Plus ` ignored? Shouldn't it be the kernel and the FreeRTOS+TCP directories that are ignored?